### PR TITLE
Projects: header banner is now short

### PR DESCRIPTION
**Pull request by breville**
screenshot 2017-07-10 14 19 25

### CODE REVIEW

**Comment by breville** [comment]
cc @tanyaparker

File: dashboard/app/views/layouts/_header.html.haml [hunk 1]
```diff
@@ -86,7 +86,7 @@
     - extra_header_banner_class = "header-banner-short"
   #header-banner{class: extra_header_banner_class, style: 'background-image: url("/shared/images/banners/teacher-homepage-hero.jpg")'}
 - if request.path == "/projects" || request.path == "/projects/public"
-  #header-banner{style: 'background-image: url("/shared/images/banners/project-banner.jpg")'}
+  #header-banner.header-banner-short{style: 'background-image: url("/shared/images/banners/project-banner.jpg")'}
```
Comment (joshlory) [inline dashboard/app/views/layouts/_header.html.haml:89]: What's the reasoning for external styles applying to React components?  Are the styles shared with the non-React headers?

**Comment by tanyaparker** [comment]
@poorvasingal confirming that you prefer to keep the header banner short (to save height) and it's okay that the image is cropped? We can update the image later (since you can no longer tell it's a phone) but I think this is an improvement for the weirdly large, empty banners.

File: dashboard/app/views/layouts/_header.html.haml (hunk 1 above)
Comment (breville) [reply dashboard/app/views/layouts/_header.html.haml:89]: Oh, this part of the header banner is part of the dashboard header and so is not built in React.

**Comment by poorvasingal** [comment]
Yes I prefer shorter banner. Thanks!!

**Pull request merged**